db/migrations/20180710115626_change_broker_catalog_descriptions_to_type_text.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table :services do
+      # rubocop:disable Migration/IncludeStringSize
```

Comment:
@Gerg Thanks for the review, we are going to discuss this comment tomorrow as a team and get back to you.

Reply:
Yes, but the problem is that there is already a difference size between mysql and postgres.
If we set the limit on a database level, that would break existing brokers with longer than 10k description.
However we agree that new fields should not use text.
We might have misinterpreted [this comment](https://www.pivotaltracker.com/story/show/157514773/comments/191012265) as an approval for ignoring the rubocop rule

Reply:
Hey @Gerg - do you have any thoughts on the above?

Reply:
cc @tcdowney and @zrob - we'd like to merge this soon as it has been open for almost a couple of months. If you could give us an idea of when this could be looked at, that would be appreciated. Thanks!

Reply:
@ablease
That makes sense. It's an existing scenario and having the brokers fail while updating (when they're probably best prepared to make changes to their descriptions) definitely sounds preferable to failing at migration time.

My only ask is that we include this reasoning as a comment next to where we are disabling RuboCop so that it's more clear.

@mattmcneeney

We should be able to address this soon. Sorry for the delay!